pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[project]
+name = "slider"
+version = "0.8.0"
+description = "Utilities for working with osu! files and data."
+authors = [{name="Joe Jevnik", email="joejev@gmail.com"}]
+license = {text="LGPLv3+"}
+readme = "README.rst"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Games/Entertainment",
+]
+requires-python = ">=3.6"
+dependencies = [
+    "numpy",
+    "requests",
+    "scipy",
+]
+scripts = {slider="slider.__main__:main"}
```

Comment:
Is this correct? I believe this will cause a [`console_scripts` entry point](https://packaging.python.org/en/latest/specifications/entry-points/) to be installed and add an executable named `slider` to the user's `PATH`. This is *not* what slider currently does: it's invoked with `python -m slider`, as on [the docs](https://llllllllll.github.io/slider/working-with-beatmaps.html#cli).